docs/source/format.rst

```diff
@@ -39,6 +39,9 @@ Many of these items allow substitutions (explained below).
   fully qualified URL (with host and scheme). If not qualified the
   test builder will be responsible for determining host and scheme.
   **Required**
+* ``query_parameters``: An optional dictionary of query parameters
+  that will be added to the ``url``. If there is an existing set of
+  query parameters they wil be extended.
```

Comment:
It's a dict like this (which is one the data structures that urlencode accepts and is the easier one to represent in yaml):

```yaml
query_parameters:
    foo: hello
    bar:
         - 1
         - 2
```

which will result in `foo=hello&bar=1&bar=2`.

I agree it is fairly confusing, thus the invitation to @FND to extend this to fix the docs as he sees fit (it was his feature request).